#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_extlink.dpatch by Milan Zamazal <pdm@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Make binutils-gold happy.

@DPATCH@
Index: speech-dispatcher/src/c/clients/say/Makefile.am
===================================================================
--- speech-dispatcher.orig/src/c/clients/say/Makefile.am	2010-06-19 09:08:14.000000000 +0200
+++ speech-dispatcher/src/c/clients/say/Makefile.am	2010-06-19 09:40:00.000000000 +0200
@@ -7,4 +7,4 @@
 
 bin_PROGRAMS = spd-say
 spd_say_SOURCES = say.c options.c options.h $(c_api)/libspeechd.h
-spd_say_LDADD = $(c_api)/libspeechd.la @EXTRA_SOCKET_LIBS@
+spd_say_LDADD = $(c_api)/libspeechd.la -lpthread @EXTRA_SOCKET_LIBS@
Index: speech-dispatcher/src/tests/Makefile.am
===================================================================
--- speech-dispatcher.orig/src/tests/Makefile.am	2010-06-19 09:08:14.000000000 +0200
+++ speech-dispatcher/src/tests/Makefile.am	2010-06-19 09:39:23.000000000 +0200
@@ -8,18 +8,18 @@
 bin_PROGRAMS = long_message clibrary clibrary2 run_test connection_recovery
 
 long_message_SOURCES = long_message.c $(c_api)/libspeechd.h
-long_message_LDADD = $(c_api)/libspeechd.la @EXTRA_SOCKET_LIBS@
+long_message_LDADD = $(c_api)/libspeechd.la -lpthread @EXTRA_SOCKET_LIBS@
 
 clibrary_SOURCES = clibrary.c $(c_api)/libspeechd.h
-clibrary_LDADD = $(c_api)/libspeechd.la @EXTRA_SOCKET_LIBS@
+clibrary_LDADD = $(c_api)/libspeechd.la -lpthread @EXTRA_SOCKET_LIBS@
 
 clibrary2_SOURCES = clibrary2.c $(c_api)/libspeechd.h
-clibrary2_LDADD = $(c_api)/libspeechd.la @EXTRA_SOCKET_LIBS@
+clibrary2_LDADD = $(c_api)/libspeechd.la -lpthread @EXTRA_SOCKET_LIBS@
 
 connection_recovery_SOURCES = connection-recovery.c $(c_api)/libspeechd.h
-connection_recovery_LDADD = $(c_api)/libspeechd.la @EXTRA_SOCKET_LIBS@
+connection_recovery_LDADD = $(c_api)/libspeechd.la -lpthread @EXTRA_SOCKET_LIBS@
 
 run_test_SOURCES = run_test.c $(c_api)/libspeechd.h
-run_test_LDADD = $(c_api)/libspeechd.la @EXTRA_SOCKET_LIBS@
+run_test_LDADD = $(c_api)/libspeechd.la -lpthread @EXTRA_SOCKET_LIBS@
 
 EXTRA_DIST= *.test *.wav
